### DISCUSSION TITLE
Miboxer FUT089Z

### DISCUSSION THREAD

**Discussion by leciuk** [Koenkk/zigbee2mqtt#9754] [Devices]
![miboxer-rgbcct-remote-zigbee-30-fut089z](https://user-images.githubusercontent.com/71944394/142689305-69623afc-a7e1-492f-a544-41da43b8fdca.jpg)

I bought this seemingly awesome zigbee 3.0 remote.
I was in no way able to connect it with zigbee2mqtt.
Is there any support?
Thanks

**Comment by rubenverhoef** [comment]
Any update on this?

I think you can put it in pairing mode by holding 'on' and 'off' buttons pressed.
See the manual: https://www.futlight.com/light/xqy/fut089z.html

But there is not yet support dor this remote, you have to make it yourself (or someone else):

https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#instructions

**Comment by type-rke** [comment]
Hi, Also following this,.

i was able to pair with ZHA, but looking for a blueprint.

**Reply by rubenverhoef** [reply]
Related: https://github.com/Koenkk/zigbee2mqtt/issues/10708

We first need to support the device. It is a TuYa device but i did not get anything from it.

**Comment by T-vK** [comment]
Basic support has been added, you can tell the remote to join by holding the on and off buttons for 10 seconds or so.

For full support, I am currently working a user extension which adds proper support for all the controls of the remote: https://github.com/Koenkk/zigbee2mqtt-user-extensions/pull/3
If you paste the code of this user extension in the extensions tab in Zigbe2MQTT, the all the controls will show up in Home Assistant. Be aware that I still have a some bugs to fix.
